KOTH/Wuhu/map.json

```diff
@@ -0,0 +1,135 @@
+{
+	"name": "Wuhu Island",
+	"authors": ["Tizona (Ported by JackJack33)"],
+	"version": "1.0.0",
+	"gametype": "KOTH",
+	"teams": [
+		{
+			"id": "blue",
+			"name": "Blue",
+			"color": "blue",
+			"max": 100,
+			"min": 1
+		},
+		{
+			"id": "red",
+			"name": "Red",
+			"color": "red",
+			"max": 100,
+			"min": 1
+		}
+	],
+
+	"spawns": [
+		{ "teams": ["spectators"], "x": -1222.5, "y": 46, "z": 459.5, "yaw": 0 },
```

Comment:
For all the spawns, it's better if you instead did "-1222.5, 46, 459.5". If the yaw is 0, you don't have to add it.